plugin.xml

```diff
@@ -84,6 +84,7 @@
         <config-file target="res/xml/config.xml" parent="/*">
             <feature name="Wechat">
                 <param name="android-package" value="xu.li.cordova.wechat.Wechat"/>
+                <param name="onload" value="true"/>
```

Comment:
This is needed for the `registerForActivityResult` bit - we have to call that so that Cordova will register this callback before the activity is started.